README.md

```diff
@@ -86,6 +86,23 @@ Afterwards, the web application will be presented on http://localhost:8000
 
 ---
 
+## How To Remove The Local Project
+
+- #### Destroy The Vagrant VM
+    Run the command:
+    ```sh
+    vagrant destroy
+    ```
+    After running the last command, the terminal will prompt the following:
+    ```sh
+    default: Are you sure you want to destroy the 'default' VM? [y/N]
+    ```
+    Type ` y `
+
+- #### Delete The Local Project Root Directory
```

Comment:
I think you can edit this to `#### Delete The Project Root Directory`.

Reply:
Sounds good, changed it